Documentation/operations/performance/tuning.rst

```diff
@@ -144,6 +144,54 @@ To validate whether your installation is running with Bandwidth Manager,
 run ``cilium status`` in any of the Cilium pods and look for the line
 reporting the status for "BandwidthManager" which should state "EDT with BPF".
 
+BBR congestion control for Pods
+===============================
+
+The base infrastructure around MQ/FQ setup provided by Cilium's Bandwidth Manager
+also allows for use of TCP `BBR congestion control <https://queue.acm.org/detail.cfm?id=3022184>`_
+for Pods. BBR is in particular suitable when Pods are exposed behind Kubernetes
+Services which face external clients from the Internet. BBR achieves higher
+bandwidths and lower latencies for Internet traffic, for example, it has been
+shown that BBR's throughput can reach as much as 2,700x higher than today's best
+loss-based congestion control; queueing delays can be 25x lower.
+
+In order for BBR to work reliably for Pods, it requires a 5.18 or higher kernel.
+As outlined in our `Linux Plumbers 2021 talk <https://lpc.events/event/11/contributions/953/>`_,
+this is needed since older kernels do not retain timestamps of network packets
+when switching from Pod to host network namespace. Due to the latter, the kernel's
+pacing infrastructure does not function properly in general (not specific to Cilium).
+We helped fixing this issue for recent kernels to retain timestamps and therefore to
```

Comment:
```suggestion
We helped fix this issue for recent kernels to retain timestamps and therefore to
```

Reply:
I think `fixing` is the correct version, no?

Reply:
I thought it was “to help do” or “to help to do”. But not a native speaker and I also had a doubt in the first place, so let's keep your version